os/include/tinyara/mpu.h

```diff
@@ -101,4 +83,10 @@ bool up_mpu_check_active(uint32_t *mpu_regs);
 
 void up_mpu_disable_region(uint32_t *mpu_regs);
 
+void mpu_regs_initialize(FAR struct mpu_usages_s *mpu);
```

Comment:
As I said before, regs could be "registers", not "regions" because of tcb_s. Let's write full word for region as you did for other function and variable.
```
void mpu_region_initialize....
```